teorija.tex

```diff
@@ -4534,10 +4540,10 @@ \subsection{Bisekcija}
   \begin{algorithmic}[1]
     \While{$|b-a| < \epsilon$}
         \State $s \gets \frac{a+b}{2}$
-        \If{\textsc{predznak}$(f(a)) =\ $\textsc{predznak}$(f(b))$}
-          \State $a \gets s$
-        \Else
+        \If{\textsc{predznak}$(f(a)) =\ $\textsc{predznak}$(f(s))$}
```

Comment:
Mislim da ni čisto prav. Definitivno je treba gledat vrednost v točki `s`, ampak potem, če sta predznaka f(a) in f(s) enaka, ves, da je nicla nekje na [s, b] in mora a postati s, ce sta pa predznaka razlicna, potem pa obstaja nicla na [a, s] in b postane s.